src/index.ts

```diff
@@ -49,6 +57,9 @@ const plugin: JupyterFrontEndPlugin<void> = {
     widget.title.caption = trans.__('Local File System');
     widget.title.icon = listIcon;
 
+    // Adding a data attribute
+    widget.node.setAttribute('data-is-filebrowser-access', '');
```

Comment:
Maybe this should be?

```suggestion
    widget.node.setAttribute('data-is-filesystem-access', '');
```